contrib/enum_metrics/enum_metrics/dbclient.py

```diff
@@ -0,0 +1,181 @@
+from cassandra.cluster import Cluster
+from cassandra.policies import (RetryPolicy)
+from cassandra.query import (PreparedStatement, BoundStatement)
+
+
+class DBClient:
+    session = None
+
+    def connect(self, nodes):
+        cluster = Cluster(
+            contact_points=nodes,
+            default_retry_policy=RetryPolicy()
+        )
+        metadata = cluster.metadata
+        self.session = cluster.connect('DATA')
+        print '\nConnected to cluster: %s' % metadata.cluster_name
+        for host in metadata.all_hosts():
+            print 'Datacenter: %s; Host: %s; Rack: %s' % (host.datacenter, host.address, host.rack)
+
+    def close(self):
+        self.session.cluster.shutdown()
+        self.session.shutdown()
+        print 'Connection closed.'
+
+    def format_results(self, results):
+        """
+        Formats results to a tuple and returns it with just the (key, column1, value) from
+        database results.
+
+        :param results:
+        :return:
+        """
+
+        return [(x.key, x.column1, x.value) for x in results]
+
+    def get_metrics_excess_enums(self, tenant_id, metric_name):
+        """
+        Retrive data from metrics_excess_enums
+        :param tenant_id:
+        :param metric_name:
+        :return:
+        """
+
+        key = tenant_id + '.' + metric_name
+        return self.__get_metrics(key, "metrics_excess_enums")
+
+    def get_metrics_preaggregated_full(self, key):
+        """
+        Retrieve data from metrics_preaggregated_full
+
+        :param key:
```

Comment:
I would prefer you either complete the param/return comments, or remove them.  To have them half done like this is distracting.